db/seeds/development.rb

```diff
@@ -96,3 +96,14 @@
                                                   access_token_audience: 'http://localhost:4000',
                                                   access_token_duration: vaid_access_token_duration,
                                                   certificates: [vaid_certificate])
+
+# Create Service Account Config for Chatbot
+chatbot = SignIn::ServiceAccountConfig.find_or_initialize_by(service_account_id: 'chatbot')
+chatbot.update!(
+  service_account_id: 'chatbot',
+  description: 'Chatbot',
+  scopes: ['http://localhost:3000/v0/map_services/token'],
+  access_token_audience: 'http://localhost:4000',
```

Comment:
@eubiera Can you give a good 'audience' for the localhost definition for the Chatbot service-to-service communication (this would be a URL for the chatbot to be calling vets-api backend from. since this is only for a local configuration, it's fine if it's just a `localhost` with a `port`, or really anything

Reply:
I think we can safely set this to http://localhost:3001/